File: RobotBokning/Models/Booking.cs
namespace RobotBokning.Models
{
    public class Booking
    {
        public int Id { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public string Description { get; set; }
        public BookingStatus Status { get; set; }

        // Relationships
        public string UserId { get; set; }
        public ApplicationUser User { get; set; }

        public int RobotId { get; set; }
        public Robot Robot { get; set; }
    }
}

File: UI/src/types/booking.ts
export interface Booking {
    id: number;
    userId: string;
    startTime: string;
    endTime: string;
    status: 'Scheduled' | 'Completed' | 'Cancelled';
  }
  
  export interface CreateBookingDto {
    startTime: string;
    endTime: string;
  }